pkg/controller/controller.go

```diff
@@ -10,6 +10,8 @@ import (
 	"errors"
 	"time"
 
+	"k8s.io/client-go/tools/record"
+
```

Comment:
I'd delete the extra line here and merge with the block below and let go reorder